hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/SCMNodeManager.java

```diff
@@ -221,9 +221,8 @@ public VersionResponse getVersion(SCMVersionRequestProto versionRequest) {
     return VersionResponse.newBuilder()
         .setVersion(this.version.getVersion())
         .addValue(OzoneConsts.SCM_ID,
-            this.scmManager.getScmStorageConfig().getScmId())
-        .addValue(OzoneConsts.CLUSTER_ID, this.scmManager.getScmStorageConfig()
-            .getClusterID())
+            this.scmStorageConfig.getScmId())
+        .addValue(OzoneConsts.CLUSTER_ID, this.scmStorageConfig.getClusterID())
```

Comment:
Minor: We can use clusterID which is set in SCMNodeManager constructor. Or remove the clusterID and use that from scmStorage. I feel 2nd option will be good, as to get ScmId we got from scmStorageConfig

Reply:
Good point, thanks. ClusterId field is removed in 50420c21c8d.